build.sbt

```diff
@@ -1,6 +1,6 @@
 import sbt.Keys._
 
-crossScalaVersions := Seq("2.11.11", "2.12.2")
+crossScalaVersions := Seq("2.11.11", "2.12.4")
```

Comment:
Shouldn't we get rid of 2.11 already? :)

Reply:
Well... We can, but why? ;-)

Reply:
It's old already :) Prepare for the future. But it's up to you.